components/gaugeVoting/gaugeVoting.js

```diff
@@ -69,6 +69,51 @@ export default function GaugeVoting({ project }) {
     return !gauge || gauge.nextVoteTimestamp === 0 || gauge.nextVoteTimestamp <= moment().unix();
   };
 
+  //10 days cooldown
+  function Countdown() {
```

Comment:
This change is not relevant to the fix mentioned in the PR. Maybe a problem during merge or rebase